modules/acquia_cms_common/acquia_cms_common.install

```diff
@@ -158,4 +158,25 @@ function acquia_cms_common_content_model_role_presave_alter(RoleInterface &$role
         break;
     }
   }
+  if (\Drupal::moduleHandler()->moduleExists('captcha')) {
```

Comment:
@deepakmishra2 can we do something like this?
 $modules_permission = [
    'shield' => 'administer shield',
    'honeypot' => 'administer honeypot',
    'captcha' => 'administer CAPTCHA settings',
    'recaptcha' => 'administer recaptcha',
  ]
  switch ($role->id()) {
    case 'user_administrator':
      foreach ($modules_permission as $module => $permission) {
        if (\Drupal::moduleHandler()->moduleExists($module)) {
          $role->grantPermission($permission);
        }
      }
    break;
  }

Reply:
updated these changes